fbbt/src/ontology/fbbt-edit.obo

```diff
@@ -68508,6 +68513,43 @@ synonym: "mNSC" NARROW [FlyBase:FBrf0184236]
 intersection_of: FBbt:00005130 ! neurosecretory neuron
 intersection_of: has_soma_location FBbt:00005802 ! pars intercerebralis
 
+[Term]
+id: FBbt:00047209
+name: embryonic precursor of scutellar pulsatile organ
+def: "An Even-skipped (Eve) positive pericardial cell that will differentiate into a scutellar pulsatile organ progenitor during late embryonic development. These are found as two pairs of two cells in each of parasegments four and five (eight cells in total)." [FlyBase:FBrf0205292]
+synonym: "embryonic precursor of scutellar pulsatile organ" EXACT []
+synonym: "embryonic precursor of wing heart" EXACT [FlyBase:FBrf0205292]
+synonym: "wing heart precursor" EXACT [FlyBase:FBrf0205292]
+is_a: FBbt:00005678 ! embryonic pericardial cell
+is_a: FBbt:00007008 ! somatic precursor cell
+relationship: expresses http://flybase.org/reports/FBgn0000606
+relationship: part_of FBbt:00047211 ! scutellar pulsatile organ anlage
+created_by: CP
+creation_date: 2018-05-31T09:42:05Z
+
+[Term]
+id: FBbt:00047210
+name: scutellar pulsatile organ progenitor
+def: "Progenitor cell specified by loss of tinman expression from embryonic precursor of scutellar pulsatile organ cells. Two pairs of cells in each of the second and third thoracic segments interconnect with cytoplasmic extensions to form a ladder-like structure anterior to the heart. In late larval stages, these cells begin to divide. In the pupa, they retract into a single cluster, split and then enter the scutellum to form the scutellar pulsatile organs." [FlyBase:FBrf0205292]
+synonym: "wing heart progenitor" EXACT [FlyBase:FBrf0205292]
+is_a: FBbt:00007008 ! somatic precursor cell
+relationship: develops_directly_from FBbt:00047209 ! embryonic precursor of scutellar pulsatile organ
+relationship: part_of FBbt:00000052 ! embryo
+relationship: part_of FBbt:00001727 ! larva
+relationship: part_of FBbt:00002953 ! pupa
```

Comment:
Slightly uncomfortable with this, but can see why you've done it.  Think OK go with this solution - at least for now.

Reply:
Yeah, I am uncomfortable with it too, I was going with convention from embryonic/larval.